Examples/Python/GroomUtils.py

```diff
@@ -457,13 +457,14 @@ def MeshesToVolumesUsingImages(outDir, meshList, imgList, printCmd=True):
             print("CMD: " + " ".join(execCommand))
         subprocess.check_call(execCommand)
 
+        spacing_string = str(img.spacing()[0]).replace(".0","")
```

Comment:
We are taking here the x-dim spacing. What if the input image has an anisotropic spacing?

Reply:
The `GenerateBinaryAndDTImagesFromMeshes` command currently automatically names the output image based on the x-spacing. It correctly rasterizes with any spacing you give it, but it only includes the x in the output name which we are renaming in the script anyway. I have already spoken with Archana about how when this command gets consolidated, we will allow the user to specify the output name to simplify things.